include/constants/vars.h

```diff
@@ -264,8 +264,8 @@
 #define VAR_ROXANNE_CALL_STEP_COUNTER                    0x40F4
 #define VAR_SCOTT_BF_CALL_STEP_COUNTER                   0x40F5
 #define VAR_RIVAL_RAYQUAZA_CALL_STEP_COUNTER             0x40F6
-#define VAR_UNUSED_0x40F7                                0x40F7 // Unused Var
-#define VAR_UNUSED_0x40F8                                0x40F8 // Unused Var
+#define VAR_DEXNAV_SPECIES                               0x40F7 // Registered dexnav species
+#define VAR_DEXNAV_STEP_COUNTER                          0x40F8 // steps for finding hidden pokemon
```

Comment:
the dexnav doesn't function properly without them, so I hesitate to to define them with default 0 values that do nothing when not set properly, but don't like the idea of defining variable values outside of vars.h

Reply:
I'm not sure why we would treat these defines different for similar ones used for terrain, wild double battles, and the vs seeker, but open to being told I'm wrong in thinking they're similar

Reply:
Quoting from Discord for posterity:

> I would personally make them defines as well, since forcing flags to be used by people not planning on using dexnav is against expansion protocol, but you can easily add static asserts that make sure there are actual flags attached to those configs if the dexnav config is enabled